README.md

```diff
@@ -162,6 +162,12 @@ This produces the preferred choices of 'Jersey' and 'United Kingdom', and regula
 By default the `country_choices` is empty, which means all countries are included, as is `preferred_country_choices`.
 The option `country_placeholder` can be specified to create a placeholder option on above the whole list.
 
+The option `country_display_type` can be specified to change the country dropdown label format. There is three formats availables :
```

Comment:
```suggestion
The option `country_display_type` can be specified to change the country dropdown label format. There are two formats available :
```

You suggest only 2 formatting for the country name, isn't it? 🤔

Reply:
In the first place, i've planned to have 3 formats. But the third, only the country code (eg. +33) does not work because a region code can be used by many regionCode.